src/psyclone/tests/psyir/transformations/intrinsics/matmul2code_trans_test.py

```diff
@@ -513,12 +513,13 @@ def test_validate11():
     vector.children[0] = my_index
     with pytest.raises(TransformationError) as excinfo:
         trans.validate(matmul)
-    assert ("Transformation Error: To use matmul2code_trans on matmul, "
-            "one or two indices of the 2nd argument 'y' "
-            "must be full ranges but found 0." in str(excinfo.value))
+    assert (f"Transformation Error: To use {trans.name} on matmul, "
```

Comment:
Again, no f-string here please.

Reply:
Addressed in c4d60bc3ef52a6d7b2853dd5b3c3b01ce82996dc.